src/ui/components/FocusContextReduxAdapter.tsx

```diff
@@ -24,6 +34,14 @@ export default function FocusContextReduxAdapter({ children }: PropsWithChildren
     });
   }, [focusRegion, loadedRegions]);
 
+  const rangeForAnalysis = useMemo(
+    () =>
+      deferredFocusRegion
+        ? { begin: deferredFocusRegion.begin.point, end: deferredFocusRegion.end.point }
+        : { begin: "0", end: endpoint },
+    [deferredFocusRegion, endpoint]
+  );
```

Comment:
This just a convenience value? I guess that makes sense, since we're using it so many places.

Reply:
> This just a convenience value? I guess that makes sense, since we're using it so many places.

Exactly. I had put it in a hook at first but then I thought that it makes sense to add it to the `FocusContext` :shrug: